lifecycle_event.go

```diff
@@ -0,0 +1,24 @@
+package cyclist
+
+import "time"
+
+var (
+	standardFluxCapacitorTime, _ = time.Parse(time.RFC3339, "1955-11-05T11:05:55-09:00")
+)
+
+type lifecycleEvent struct {
+	Event     string    `json:"event"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+func newLifecycleEvent(event, ts string) *lifecycleEvent {
+	timestamp, err := time.Parse(time.RFC3339Nano, ts)
+	if err != nil {
+		timestamp = standardFluxCapacitorTime
```

Comment:
What I think I meant to say, but was too tired/jet-lagged to articulate is: If someone sees this in a response unexpectedly, do you think it would be obvious that it's a "sentinel" date that means there's some error/no date, or would `nil` be better? I'm not really familiar enough with where these timestamps appear to be able to vote one way or the other.